.github/workflows/stats_merged_prs.yml

```diff
@@ -0,0 +1,25 @@
+name: Merged PR stats
+
+on:
+  pull_request:
+    branches:
+      - main
+      - v*-branch
+    types: [closed]
+jobs:
+  record_merged:
+    if: github.event.pull_request.merged == true && github.repository == 'zephyrproject-rtos/zephyr'
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: PR event
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ELASTICSEARCH_KEY: ${{ secrets.ELASTICSEARCH_KEY }}
+          ELASTICSEARCH_SERVER: "https://elasticsearch.zephyrproject.io:443"
+        run: |
+          pip3 install pygithub elasticsearch
+          python3 ./scripts/ci/stats/merged_prs.py --pull-request ${{ github.event.pull_request.number }} --repo zephyrproject-rtos/zephyr
```

Comment:
`${{ github.repository }}` instead of the hardcoded repository?

Reply:
fixed